domains/beatrack.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "UsernameKhoaIsAvailable",
+           "email": "khoatm20@gmail.com",
+           "discord": "697058629517443106"
+        },
+    
+        "record": {
+            "A": ["192.168.1.200"]
```

Comment:
This is a local IP, please put a public IP here

Reply:
I need to make another registration or not

Reply:
> I need to make another registration or not

You can just edit the json file in your fork